validator/sequences/feature_enumeration/sequence.md

```diff
@@ -1,5 +1,5 @@
 
-## feature_enumeration (ALPHA)
+## feature_enumeration (BETA)
```

Comment:
Our current criteria for BETA says a manufacturer must have implemented it. Has any yet? Maybe the requirement should be flexible

Reply:
Yah -- I was thinking about this.  One option might be to have a "PREVIEW" stage or something that says "this is not Alpha anymore but not quite BETA" -- which would be that interstitial phase of "we're working with somebody on this". So the progression would be ALPHA -> PREVIEW -> BETA -> STABLE... thoughts?  Easy enough to add in to the code!